contentify/Controllers/InstallController.php

```diff
@@ -757,6 +759,7 @@ public function createDatabase()
         {
             $table->boolean('open')->default(true);
             $table->boolean('internal')->default(false);
+            $table->boolean('enable_comments')->default(false);
```

Comment:
Why `false` here?

Reply:
"Idea" was that you most likely don't want comments enabled by default on newly created "Pages".

Reply:
Now I am worried, this is in the polls table schema, not in the pages?

Reply:
I'll fix! Guess it went a bit fast here.. Wanted to correct my error :(

Reply:
It **_is_** the current default: https://github.com/Contentify/Contentify/blob/v3.0-alpha/contentify/Controllers/InstallController.php#L409 :P

Reply:
But be aware that there is also the default value of the checkox in the form template. Which is set to `true`: https://github.com/Contentify/Contentify/blob/v3.0-alpha/app/Modules/Pages/Resources/Views/admin_form.blade.php#L20

Reply:
> It **_is_** the current default: https://github.com/Contentify/Contentify/blob/v3.0-alpha/contentify/Controllers/InstallController.php#L409 :P

Yeah, I meant to ask if you were fine/agreed with it being defaulted as false 😛

Reply:
One last thing: Should I close the pull request, and open a new if I want to add more later on? 
rather than having all my small changes get posted to this one?

Reply:
It's okay to post all cahnges in rhis PR.